### PR TITLE
LB listener remove fix: Update host-iptables config on port.deactivate

### PR DESCRIPTION
**Pull request by alena1108**
https://github.com/rancherio/rancher/issues/431

### CODE REVIEW

**Comment by alena1108** [comment]
@sonchang @cjellick Son, Craig, can you please review this PR? Merging it in will help QA in their testing as now they have to clear up obsolete LB listeners ports from iptables manually.

**Comment by sonchang** [comment]
LGTM

File: code/iaas/lb-instance/src/main/java/io/cattle/platform/lb/instance/process/LoadBalancerUpdateConfig.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package io.cattle.platform.lb.instance.process;
 
-import static io.cattle.platform.core.model.tables.PortTable.*;
+import static io.cattle.platform.core.model.tables.PortTable.PORT;
```
Comment (cjellick) [inline code/iaas/lb-instance/src/main/java/io/cattle/platform/lb/instance/process/LoadBalancerUpdateConfig.java]: FYI, Darren has a preference for dong * for any number of static imports.
He had me change `Java > Code Style > Organize Imports > Number of static imports needed...` to 1.
Just letting you know; no need to change this instance.

**Comment by cjellick** [comment]
LGTM

**Pull request merged**